meteor/lib/collections/RundownPlaylists.ts

```diff
@@ -35,51 +35,70 @@ export type RundownPlaylist = DBRundownPlaylist
  * These used to reside on the Rundown class
  */
 export class RundownPlaylistCollectionUtil {
-	/** Returns all Rundowns in the RundownPlaylist */
-	static getRundowns(
-		playlist: Pick<RundownPlaylist, '_id'>,
+	static getRundownsOrdered(
+		playlist: Pick<RundownPlaylist, '_id' | 'rundownIdsInOrder'>,
 		selector?: MongoQuery<Rundown>,
 		options?: FindOptions<Rundown>
 	): Rundown[] {
-		return Rundowns.find(
+		const allRundowns = Rundowns.find(
```

Comment:
DRY: This should be a call to `RundownPlaylistCollectionUtil.getRundownsUnordered(playlist, selector, options)`